sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetScan.scala

```diff
@@ -774,9 +774,11 @@ private case class GpuParquetFileFilterHandler(
           val clipped = GpuParquetUtils.clipBlocksToSchema(clippedSchema, blocks, isCaseSensitive)
           (clipped, clippedSchema)
         }
-
+      val hasDateTimeInReadSchema = DataTypeUtils.hasDateOrTimestampType(readDataSchema)
```

Comment:
nit: There is an `isOrContainsDateOrTimestamp` in `GpuOverrides`. Can we use it?
https://github.com/NVIDIA/spark-rapids/blob/c92bbc0c412a5b2245638263ac6b0d9d4b19f28b/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala#L712-L713

Reply:
Yes, we can use it. Since it's a nit, let's merge this PR first.